feature-group-definitions/webdriver-bidi.dist.yml

```diff
@@ -0,0 +1,19 @@
+# Generated from: webdriver-bidi.yml
+# Do not edit this file by hand. Edit the source file instead!
+# Last modified: 2024-04-04T09:37:18.561438534+00:00[UTC]
+
+name: WebDriver BiDi
+spec: https://w3c.github.io/webdriver-bidi/
+# WebDriver BiDi is not in BCD, see https://github.com/mdn/browser-compat-data/issues/20207
+# The browser releases listed here are from:
+# https://developer.chrome.com/blog/webdriver-bidi-2023
+# https://bugs.chromium.org/p/chromedriver/issues/detail?id=4016
+# https://bugzilla.mozilla.org/show_bug.cgi?id=1753997
+status:
```

Comment:
This file demonstrates preserving comments in the original file.

Reply:
I thought this would be hard/impossible, but here it is, it works!